### DISCUSSION TITLE
How do you install same feature set for both target and host triplet?

### DISCUSSION THREAD

**Discussion by funnan-dev** [microsoft/vcpkg#40413] [Q&A]
I am trying to use a minimal set of features with `qtbase` to reduce the size of artifact storage and docker layers by defining the following manifest:

```json
{
  "name": "project",
  "version-string": "2023.10.22",
  "port-version": 1,
  "description": "Project",
  "homepage": "https://project.com",
  "dependencies": [
    {
      "name": "qtbase",
      "default-features": false,
      "features": [
        "png"
      ]
    },
    "qtdeclarative",
    "gtest"
  ]
}
```

Doing this works great when I define `VCPKG_TARGET_TRIPLET=x64-linux` and `VCPKG_HOST_TRIPLET=x64-linux` to be the same triplet `x64-linux`.

I get the following: 
```shell
qtbase[concurrent,core,doubleconversion,freetype,gui,network,png,testlib,thread]:x64-linux@6.7.2
```

The problem occurs when I try to cross-compile for android using `VCPKG_TARGET_TRIPLET=x64-android` and `VCPKG_HOST_TRIPLET=x64-linux`.

I get the following: 

```shell
  * qtbase[brotli,concurrent,core,dbus,default-features,doubleconversion,egl,fontconfig,freetype,gui,harfbuzz,icu,jpeg,network,opengl,openssl,pcre2,png,sql,sql-psql,sql-sqlite,testlib,thread,widgets,xcb,xcb-xlib,xkb,xkbcommon-x11,xlib,xrender,zstd]:x64-linux@6.7.2
    qtbase[concurrent,core,doubleconversion,freetype,gui,network,png,testlib,thread]:x64-android@6.7.2
```

Is there a way to set the host triplet's `"default-features": false`? I **really** do not need Qt's `widget,sql,sql-psql,sql-sqlite`, etc... in my pipelines using up storage and time during my CI/CD pipelines.

**Comment by dg0yt** [comment]
> Is there a way to set the host triplet's `"default-features": false`?

Try this additional dependency:
~~~json
    {
      "name": "qtbase",
      "default-features": false,
      "host": true
    },
~~~
